src/test/java/com/wikia/webdriver/testcases/adstests/TestInterstitial.java

```diff
@@ -23,10 +22,39 @@ public void interstitialAdScaled(
       String wikiName,
       String article,
       Dimension pageSize,
-      double scaledAdRatio
+      Dimension adSize,
+      boolean isAdShouldBeScaled
+  ) {
+    testInterstitial(wikiName, article, pageSize, adSize, isAdShouldBeScaled);
+  }
+
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      groups = "InterstitialMobile",
+      dataProvider = "interstitialMobile"
+  )
+  public void interstitialAdScaledMobile(
```

Comment:
Do we really need this wrapper-method? Can't we make `testInterstitial()` our test method which gets data from the data provider?